Client/Frontend/Browser/Tab.swift

```diff
@@ -123,7 +123,7 @@ class Tab: NSObject {
 
     /// Whether or not the desktop site was requested with the last request, reload or navigation. Note that this property needs to
     /// be managed by the web view's navigation delegate.
-    var desktopSite: Bool = false
+    var desktopSite: Bool = UIDevice.isIpad ? true : false
```

Comment:
This should be a `Preference` that can be set in settings. Turned on by default on iPads.

I've started some initial work on it here if you want to take a look https://github.com/brave/brave-ios/compare/development...iccub:bugfix/1290?expand=1